pkg/daemon/update.go

```diff
@@ -734,6 +755,16 @@ BOOT_ID=%s
 PENDING=%d`, pendingStateMessageID, pending.GetName(), dn.bootID, isPending)))
 
 	logger.Stdin = &pendingState
+	loggerOutput, err := logger.CombinedOutput()
+	if err != nil {
+		// RHEL 7.6 logger (util-linux) doesn't have the --journald flag and it's failng to log the above.
+		if !strings.Contains(string(loggerOutput), "unrecognized option '--journald'") {
+			return loggerOutput, err
+		}
+		// Super hacky workaround for the above
+		oldLogger := exec.Command("logger", fmt.Sprintf(`{"MESSAGE": "%s", "BOOT_ID": "%s", "PENDING": "%d", "OLD_LOGGER": "1"}`, pending.GetName(), dn.bootID, isPending))
```

Comment:
Then in this code we wouldn't try and fall back each time, we'd just do:

```
if (!dn.hostRhel7) {
   logger --journald
} else {
  logger with literal JSON as message
}
```

Reply:
see https://github.com/openshift/machine-config-operator/pull/734#discussion_r282886553